graylog-storage-elasticsearch7/src/main/java/org/graylog/storage/elasticsearch7/MoreSearchAdapterES7.java

```diff
@@ -62,7 +61,7 @@
 
 public class MoreSearchAdapterES7 implements MoreSearchAdapter {
     private static final Logger LOG = LoggerFactory.getLogger(MoreSearchAdapterES7.class);
-    public static final IndicesOptions INDICES_OPTIONS = IndicesOptions.fromOptions(false, false, true, false);
+    public static final IndicesOptions INDICES_OPTIONS = IndicesOptions.fromOptions(true, true, true, false);
```

Comment:
Could you replace it with `IndicesOptions.LENIENT_EXPAND_OPEN` which is already present in other places, so that we have the same method of options creation?

Reply:
I started doing this and realized that we actually use `fromOptions` a lot more than the constant values.
The constants are more declarative, but it's not obvious whether `allowNoIndices` is true or false. Also, it's more obvious what is being changed by this PR, if we stick with the booleans.

Reply:
OK - replaced with `LENIENT_EXPAND_OPEN` or `LENIENT_EXPAND_OPEN_CLOSED` wherever possible.